oshi-core/src/test/java/oshi/software/os/FileSystemTest.java

```diff
@@ -33,6 +33,8 @@ void testFileSystem() throws IOException {
                 is(true));
         assertThat("File system max open file descriptors should be 0 or higher",
                 filesystem.getMaxFileDescriptors() >= 0L, is(true));
+        assertThat("File system max open file descriptors per process should be 0 or higher",
+                filesystem.getMaxFileDescriptorsPerProcess() >= 0L, is(true));
```

Comment:
I realize you copied the one above, but both of these should be `greaterThanOrEqualTo(0)`

Reply:
Missed one :)